src/products/components/ProductMediaPage/ProductMediaPage.tsx

```diff
@@ -122,15 +125,27 @@ const ProductImagePage: React.FC<ProductImagePageProps> = props => {
               <Card>
                 <CardTitle
                   title={intl.formatMessage({
-                    defaultMessage: "Photo View",
+                    defaultMessage: "Media View",
                     description: "section header"
                   })}
                 />
                 <CardContent>
-                  {!!image ? (
-                    <div className={classes.imageContainer}>
-                      <img src={image.url} className={classes.image} />
-                    </div>
+                  {!!mediaObj ? (
+                    mediaObj?.type !== ProductMediaType.IMAGE ? (
```

Comment:
Let's avoid doing a ? b ?  c : d - it's get's very difficult to catch what does what when not familiar with the code